webpack-demo/src/listFunctions.js

```diff
@@ -0,0 +1,93 @@
+// eslint-disable-next-line import/no-cycle
+import { showList } from './index.js';
+
+// eslint-disable-next-line import/no-mutable-exports
+export let listData = [{}];
+if (localStorage.toDoStorage !== undefined) {
+  listData = JSON.parse(localStorage.toDoStorage);
+}
+function displayTasks() {
+  showList.innerHTML = '';
+  listData.forEach((toDoArr, index) => {
+    const li = document.createElement('li');
+    li.classList.add('oneTask');
+    const p1 = document.createElement('p');
+    p1.id = 'indexNum';
+    p1.innerText = toDoArr.index;
+    li.appendChild(p1);
+
+    const p2 = document.createElement('p');
+    p2.innerText = toDoArr.description;
+    li.appendChild(p2);
+
+    const label = document.createElement('label');
+    label.setAttribute('for', 'checkbox');
+    li.appendChild(label);
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.id = 'checkbox';
+    li.appendChild(checkbox);
+
+    const button = document.createElement('button');
+    button.classList.add('threeDots');
+    button.innerText = '__';
+    li.appendChild(button);
+    button.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Kindly make sure to refrain from using eslint disable as it defies the purpose of using linters in the first place.